docs/content/how-to/distribute-hosted-cluster-workloads.md

```diff
@@ -28,6 +28,41 @@ In addition:
       role.kubernetes.io/infra: "" 
 ```
 
+## Scheduling Topology Options 
+
+Cluster Service Providers may choose how hosted control planes are isolated or co-located. The three different options are:
+
+ - Shared Everything
+ - Shared Nothing 
+ - Dedicated Request Serving
+
+**NOTE**: Each hosted control plane can run Single Replica or Highly Available. If Highly Available, the control plane will be spread across failure domains via `topology.kubernetes.io/zone` as the topology key.
+
+### Shared Everything
+
+- All hosted control plane pods are scheduled to any node that can run hosted control plane workloads.
+
+![shared_everything_topology.png](..%2Fimages%2Fshared_everything_topology.png)
+
+### Shared Nothing
+
+- Nodes meant for a specific hosted cluster are tainted (and labeled) with a specific `hypershift.openshift.io/cluster` value.
+- No other control plane pods will land on those nodes.
+
+![shared_nothing_topology.png](..%2Fimages%2Fshared_nothing_topology.png)
+
+### Dedicated Request Serving
+
```

Comment:
It's also worth mentioning that in this topology, only 2 replicas (as opposed to 3) of request serving workloads are deployed.

Reply:
Ive added a point about this but why is this the case in HA?